### DISCUSSION TITLE
Who's better? Windows 11 or Windows 10?

### DISCUSSION THREAD

**Discussion by Pixel556** [DustinBrett/daedalOS#171] [Polls]
Answer the poll.

**Comment by Pixel556** [comment]
I prefer Windows 10.

**Comment by DustinBrett** [comment]
For looks, at the moment I still prefer Windows 10. I feel it's got a minimalism to it that I like. Before that I used to like Windows 2000. In between I usually tried to make other versions of Windows looks like what I remembered from Win 2k. But now I am happy to stick with 10 and if I went to 11 I would probably change it to look like 10.

As for the OS itself, I have no clue tbh because I have not used Windows 11. If I were to believe the articles I see it seems like it has some kinks to work out. But I am familiar with that and Windows, seems like the "Service packs" were always better back in the day.

I will eventually go to Windows 11 or perhaps skip it and jump on 12 or who knows what they got up their sleeves (No I). I'm in no rush to leave Windows 10 as it does everything I want.

**Reply by Pixel556** [reply]
You could use ExplorerPatcher to get back the 10 taskbar on 11. It also supports 10.

**Reply by DustinBrett** [reply]
Ya I used to mod Windows back in the 98 days even. But now-a-days I liked to try and get what I can without adding more programs or tools, or patching something in a way that makes updates harder.
